test/libponyc/codegen.cc

```diff
@@ -14,7 +14,13 @@
 
 
 class CodegenTest : public PassTest
-{};
+{
+  virtual void SetUp()
+  {
+    PassTest::SetUp();
+    set_builtin(NULL);
```

Comment:
I'd recommend putting this in the new test instead, before `TEST_COMPILE`. It would avoid having to compile the real builtin for every `CodegenTest` test and instead compile it only for the test that needs it, avoiding an increase in execution time for every test.